docs/SemanticARC.md

```diff
@@ -194,7 +244,11 @@ In this case, since the apply's second argument must be @owned, a simple use-def
       return %result : $()
     }
 
-While this may look correct to the naked eye, it is actually incorrect even in SIL today. This is because in SIL today, switch_enum always takes arguments at +1! Yet, in the IR there is no indication of the problem and the code will compile! Now let us update the IR given semantic ARC:
+While this may look correct to the naked eye, it is actually incorrect even in SIL today. This is because in SIL today, switch_enum always takes arguments at +1!
+
+[EE] Hm... no. switch_enum doesn't take it's argument as owned.
```

Comment:
@eeckstein I am not saying that that is how things are today, but from what I understand that is the model that we are supposed to be following.

@rjmccall, @jckarter I think this came up in conversation before. Thoughts?